mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/services/AvtaleService.kt

```diff
@@ -44,15 +44,19 @@ class AvtaleService(
     suspend fun upsert(request: AvtaleRequest, navIdent: String): Either<List<ValidationError>, AvtaleAdminDto> {
         virksomhetService.getOrSyncVirksomhet(request.leverandorOrganisasjonsnummer)
 
+        val previous = avtaler.get(request.id)
         return validator.validate(request.toDbo())
             .map { dbo ->
                 db.transaction { tx ->
                     avtaler.upsert(dbo, tx)
                     utkastRepository.delete(dbo.id, tx)
 
-                    dispatchNotificationToNewAdministrators(tx, dbo, navIdent)
-
                     val dto = getOrError(dbo.id, tx)
+                    if (previous == dto) {
```

Comment:
`updatedAt` har blitt fjernet fra dto'en for at denne sjekken skal funke, men ender vi nå opp med at `updated_at`-kolonnen likevel blitt endret etter at `upsert` ble kjørt på linje 51 over her, selv om det egentlig ikke har skjedd en endring på selve avtalen?
Lurer i så fall på om vi skulle revurdert å legge til en `getDbo()`-metode for avtaler/gjennomføringer og sammenligne disse før vi kjører `upsert` i stedet for etter. Tanker?

Reply:
Jo, det er mulig. Testa å gjøre det sånn nå, blir ikke så værst. Fjernet avslutningsstatus fra dbo, da den alltid er IKKE_AVSLUTTET (men konvertert fra dto er den ikke alltid det). For arena dbo'en burde denne være riktig, dvs dbo skal ha AVSLUTTET når sluttdato er passert, og dto -> dbo gir AVSLUTTET, så burde ikke være noe problem der.

Det får meg til å ville fjerne avslutningsstatus fra DB som vi har snakket om egentlig. Heller ha et avbrutt tidspunkt. Så kan vi sette alle med status AVBRUTT til avbrutt_tidspunkt = start_dato og AVLYST til avbrutt_tidspunkt = start_dato - 1 f. eks.